src/apps/search-header/search-header.tsx

```diff
@@ -98,12 +96,17 @@ const SearchHeader: React.FC = () => {
 
   // downshift prevents the default form submission event when the autosuggest
   // is open - that's why in some cases we have to simulate form sumbission
-  function manualRedirect(inputValue: string) {
-    const baseUrl = t("searchHeaderUrlText");
-    const params = inputValue;
-    if (window.top) {
-      window.top.location.href = `${baseUrl}?q=${params}`;
+  function manualRedirect(itemValue: Suggestion) {
+    const inputValue = itemToString(itemValue);
+    if (!window.top) {
+      return;
+    }
+    // if this item is one of the work suggestions, redirect to material page
+    if (materialData.includes(itemValue)) {
+      window.top.location.href = `${baseUrl}/work/${itemValue.work?.manifestations.first.pid}`;
```

Comment:
Please clear this path with @spaceo .

I think it is a mistake. The data we are showing on the work page is a work - not a material. If that is is the case the path should be generated from `itemValue.work?.workId`.

Reply:
Mmmmm but it actually is a material. Try to check material.tsx file - the query is called useGetMaterial() and it accepts a pid, as opposed to workId. 
The reason for this that there always has to be one specific material selected (availability labels on the top left part of the material header)
![image](https://user-images.githubusercontent.com/28546954/183454859-30f24bef-98b4-4081-baed-42ec980f196c.png) But I'll talk to Mikkel tomorrow ;)